webpack.config.js

```diff
@@ -76,5 +78,11 @@ module.exports = {
       filename: 'index.html',
       inject: 'body',
     }),
+    new UglifyJSPlugin(),
+    new webpack.DefinePlugin({
```

Comment:
this way you will always run your webpack in production environment, are you sure you want that?